todo/src/components/Todo.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { toggleCompleted, deleteTodo } from '../actions/actions';
+
+const Todo = ({ todo, listNumber, toggleCompleted, deleteTodo }) => {
```

Comment:
good job recognizing that you can destructure the props here like you normally could, because the state is being mapped to the props being passed in with the connect(MSTP) down at the bottom :+1: